test/unit/stateful-chainlink-oracle.spec.ts

```diff
@@ -119,11 +119,44 @@ describe('StatefulChainlinkOracle', () => {
     });
   });
 
-  describe('reconfigureSupportForPair', () => {
+  describe('isPairAlreadySupported', () => {
+    when('there is no pricing plan', () => {
+      let isAlreadySupported: boolean;
+      given(async () => {
+        await chainlinkOracle.setPricingPlan(TOKEN_A, TOKEN_B, NO_PLAN);
```

Comment:
Is it necessary to explicit this? Should this be `NO_PLAN` by default?

Reply:
It is not necessary, but I wanted to make it explicit for the tests